README.md

```diff
@@ -20,22 +20,26 @@ This image is published to Docker Hub and can be pulled via the command
 docker pull ppiper/node-browsers
 ```
 
-The default tag contains node 10 due to compatibility reasons with approuter.
+The default tag `latest` contains node 10 compatibility reasons with approuter, it is which is a Maintenance LTS and will be end of life after 2021-04-30.
 
-For node 12, use the tag `node12`:
+For node-12 or node-14, use the tag `12-buster` or `14-buster`:
 
 ```
-docker pull ppiper/node-browsers:node12
+docker pull ppiper/node-browsers:12-buster
+docker pull ppiper/node-browsers:14-buster
 ```
 
 ## Build
 
 To build this image locally, open a terminal in the directory of the Dockerfile and run
 
 ```
-docker build -t ppiper/node-browsers .
+docker build --build-arg=BASE_IMAGE_TAG=10-buster -t ppiper/node-browsers .
 ```
 
+Where the `BASE_IMAGE_TAG` build argument can be replaced with `12-buster` or `14-buster`.
```

Comment:
Ah... to avoid the confusion that I had, you could perhaps write this:
```suggestion
Where the `BASE_IMAGE_TAG=10-buster` build argument can be replaced with `BASE_IMAGE_TAG=12-buster` or `BASE_IMAGE_TAG=14-buster`.
```

My confusion was that I could replace BASE_IMAGE_TAG by 14-buster, then it would have read "--build-arg=14-buster=10-buster"